pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -361,6 +388,19 @@ type MasqueradePodInterface struct {
 	gatewayIpv6Addr     *netlink.Addr
 }
 
+func (b *MasqueradePodInterface) SetInitialPodNetworkConfig() error {
+	//if Handler.Ipv6NatEnabled() {
```

Comment:
please note that since ipv6 is still not configured at the time the method is performed, then I cant tell if ipv6 nat is enabled. consequently, ConfigureIpv6Forwarding happens with no condition. please advise.